sections/03_categorize.md

```diff
@@ -422,28 +455,29 @@ deep neural networks. The ability to discover relevant sequences of events from
 large number of trajectories requires powerful and flexible feature construction
 methods - an area at which deep neural networks tend to excel.
 
-#### Has deep learning already induced a strategic inflection point for one or more aspects?
+#### Has deep learning already induced a strategic inflection point for one or
```

Comment:
We're going to drop all the text that follows this. Feel free to delete if you want to, or I can in a separate pull request to address #352